packages/starlight/__e2e__/tabs.test.ts

```diff
@@ -0,0 +1,122 @@
+import { expect, testFactory, type Locator } from './test-utils';
+
+const test = await testFactory('./fixtures/basics/');
+
+test('syncs tabs with a click event', async ({ page, starlight }) => {
```

Comment:
I updated branch and this test is failing now. I wonder if it’s due to #1723, which added `aria-select="false"` to inactive tabs? Might need to update the logic to correctly target the active tab.

Reply:
Turns out, this was a flacky test and it took me a bit to figure out what was going on. After trying to reproduce locally and couldn't, I tried a Linux machine, a Windows machine but all worked fine.
I had to enable [traces](https://playwright.dev/docs/trace-viewer) of the test on GitHub Actions and replay it locally to figure it out (amazing feature btw) what was going on.
During the early stage of the PR, I used the dev server to setup the E2E tests to iterate faster and never thought of switching to a production build (which is what we ultimately want to test). According to the traces, during the huge amount of requests made by Vite, the page could end up being reloaded entirely sometimes on top of being slow to load the first time (I assume due to the limited resources of the CI).
After noticing this pattern in failing traces, I switched to a production build as expected and this also fixed the flacky test.